plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/edit/SQLServerViewManager.java

```diff
@@ -104,10 +104,13 @@ private void createOrReplaceViewQuery(DBRProgressMonitor monitor, DBCExecutionCo
 
         switch (action) {
             case CREATE:
-                actions.add(new SQLDatabasePersistAction("Create view", view.getDDL()));
+                actions.add(new SQLDatabasePersistAction("Create view", view.getName()));
                 break;
             case ALTER:
-                actions.add(new SQLDatabasePersistAction("Alter view", SQLServerUtils.changeCreateToAlterDDL(view.getDataSource().getSQLDialect(), view.getDDL())));
+                String viewDDL = view.getDataSource().isAtLeastV16()
+                           ? view.getDDL().replace("CREATE", "CREATE OR ALTER")
```

Comment:
This still will replace all occurrences of `CREATE`. What's wrong with `SQLServerUtils.changeCreateToAlterDDL`?

Reply:
@ShadelessFox fixed